pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -66,9 +67,11 @@ in stdenvNoCC.mkDerivation rec {
   version = version_coreboot;
   src = tar_coreboot;
 
-  nativeBuildInputs = [ perl curl gnumake git bison ];
+  nativeBuildInputs = [ perl curl gnumake bison ];
 
-  buildInputs = [ gnat11 flex zlib ];
+  buildInputs = [ flex gnat11 zlib ];
+
+  propagatedBuildInputs = [ git ncurses pkg-config python3Minimal ];
```

Comment:
Are those packages really strictly required at all time for this package to be useful? Can we maybe use makeWrapper?

Reply:
> Can we maybe use makeWrapper?

No, because as far as I can tell, these are not run-time dependencies of the code packaged here, but they are meant to be further tools that belong into the Coreboot toolchain.

Reply:
We don't propagate tools that are nice to have. This is only done if those packages are strictly required and the package is not useable without them.

Reply:
These are not nice to have. These are required so that the toolchain is fully functional. We can not use makeWrapper since we would have to wrap scripts from the coreboot build system, which is outside of this package.

Reply:
I just added libuuid since this is required to build TianoCore.